test/lib_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+NC='\033[0m'
+CYAN='\033[0;36m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+
+INPUT_FILES="lib/*.ods"
+TMP_FILE=$(mktemp "compiled.XXXXX")
+printf "${CYAN}Running compiler 'pass' tests...\n${NC}"
+
+for input_file in $INPUT_FILES; do
+    output_file=${input_file/.ods/.out}
+
+    # compile odds program to temp python file
+    ../compiler/odds.sh -c $input_file $TMP_FILE
```

Comment:
Do we want to compile or use raw? Or are we comparing the runtime output most of the time, cause i think it would be difficult to compare **actual** output of the python. This may change..